cmake/FindGDIPLUS.cmake

```diff
@@ -17,7 +17,7 @@ else()
     set(GDIPLUS_LOWERCASE 1 CACHE INTERNAL "Is GdiPlus.h spelt with lowercase?")
 endif()
 
-find_library(GDIPLUS_LIBRARY NAMES gdiplus)
+find_library(GDIPLUS_LIBRARY NAMES libgdiplus gdiplus)
```

Comment:
In MSYS2 environment, I had cmake which did not find gdiplus. Adding a suggested name libgdiplus fixed the problem